src/DependencyInjection/FOSHttpCacheExtension.php

```diff
@@ -48,7 +49,7 @@ public function load(array $configs, ContainerBuilder $container): void
 
         if ($config['debug']['enabled'] || (!empty($config['cache_control']))) {
```

Comment:
@dbu Do we want to have the `CacheControlListener` registered if only the new `ttl_header` config is defined, for maybe just the debug ENVs?

If not we would need to go with: `count($config['cache_control']) > 1` and `addDefaultsIfNotSet` in the config.

Reply:
i can't see a use case for configuring the ttl_header but not configuring any reverse_proxy_ttl at all.

if we merge as is now, my understanding is that the behaviour is the same as before, except that the ttl_header configuration is respected if overwritten, but does not cause errors if not explicitly mentioned by the user. am i missing something?

Reply:
yes it will not break, it just register a listener which does then nothing as no rules are configured.

Reply:
i clarified with alexander, we leave it like this. the cleanest way would be to count if there are any cache_control.rules, but this is not the bug and could have edge case side effects if somebody relies on the cache control service to exist even without rules if there is any configuration.

could be a cleanup for 4.x. when we count the rules, we can also addDefaultsIfNotSet on the ttl_header